app-emulation/FEX/FEX-2408.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic cmake toolchain-funcs
+
+DESCRIPTION="A fast usermode x86 and x86-64 emulator for Arm64 Linux"
+HOMEPAGE="https://fex-emu.com"
+
+JEMALLOC_HASH="7ae889695b8bebdc67c004c2c9c8d2e57748d2ab"
+JEMALLOC_GLIBC_HASH="888181c5f7072ab1bd7aa7aca6d9f85816a95c43"
+# TODO: unvendor this when this version will be available in gentoo
+FMTLIB_V="10.1.1"
+CPP_OPTPARSE_HASH="eab4212ae864ba64306f0fe87f102e66cb5a3617"
+# TODO: The following two will be hard-vendored in next release, drop them then.
+JSON_MAKER_HASH="8ecb8ecc348bf88c592fac808c03efb342f69e0a"
+TINY_JSON_HASH="9d09127f87ea6a128fb17d1ffd0b444517343f1c"
+# TODO: This would be possible to unvendor in the next release
+VIXL_HASH="a90f5d5020c305d03d3182dcc90a31321cc7a661"
+ROBIN_MAP_HASH="f1ab6900466891af11e3c264c63acf1dd9c3532c"
+# TODO: Unvendor and replace this with Qt in next release
+IMGUI_HASH="4c986ecb8d2807087fd8e34894d1e7a138bc2f1d"
+
+SRC_URI="
+	https://github.com/FEX-Emu/jemalloc/archive/${JEMALLOC_HASH}.tar.gz -> jemalloc-${JEMALLOC_HASH}.tar.gz
+	https://github.com/FEX-Emu/jemalloc/archive/${JEMALLOC_GLIBC_HASH}.tar.gz -> jemalloc-glibc-${JEMALLOC_GLIBC_HASH}.tar.gz
+	https://github.com/fmtlib/fmt/archive/refs/tags/${FMTLIB_V}.tar.gz -> libfmt-${FMTLIB_V}.tar.gz
+	https://github.com/Sonicadvance1/cpp-optparse/archive/${CPP_OPTPARSE_HASH}.tar.gz -> cpp-optparse-${CPP_OPTPARSE_HASH}.tar.gz
+	https://github.com/Sonicadvance1/json-maker/archive/${JSON_MAKER_HASH}.tar.gz -> json-maker-${JSON_MAKER_HASH}.tar.gz
+	https://github.com/Sonicadvance1/tiny-json/archive/${TINY_JSON_HASH}.tar.gz -> tiny-json-${TINY_JSON_HASH}.tar.gz
+	https://github.com/FEX-Emu/vixl/archive/${VIXL_HASH}.tar.gz -> vixl-${VIXL_HASH}.tar.gz
+	https://github.com/FEX-Emu/robin-map/archive/${ROBIN_MAP_HASH}.tar.gz -> robin-map-${ROBIN_MAP_HASH}.tar.gz
+	https://github.com/Sonicadvance1/imgui/archive/${IMGUI_HASH}.tar.gz -> imgui-${IMGUI_HASH}.tar.gz
+	https://github.com/FEX-Emu/${PN}/archive/refs/tags/${P}.tar.gz
+"
+
+S="${WORKDIR}/${PN}-${P}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="-* ~arm64"
+BDEPEND="
+	sys-devel/clang
+	sys-devel/llvm
+"
+RDEPEND="
+	dev-libs/xxhash
+	X? (
+		media-libs/libsdl2
+		media-libs/libglvnd
+		media-libs/libepoxy
+		x11-libs/libX11
+	)
+"
+DEPEND="
+	>=sys-kernel/linux-headers-6.8
+	${RDEPEND}
+"
+
+PATCHES="
+	${FILESDIR}/${P}-unvendor-xxhash.patch
+	${FILESDIR}/${P}-unvendor-drm-headers.patch
+	${FILESDIR}/${P}-tiny-json-as-static.patch
+	${FILESDIR}/${P}-fmt-as-static.patch
+	${FILESDIR}/${P}-json-maker-as-static.patch
+	${FILESDIR}/${P}-imgui-as-static.patch
+"
+
+IUSE="+lto X"
```

Comment:
These days, we tend to let the user specify it. See https://bugs.gentoo.org/937459 too.